tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -599,24 +594,26 @@ export class TestCodeModeler {
         }
     }
 
+    public createApiScenarioLoaderOption(fileRoot: string) {
```

Comment:
Does it mean the latest OAV cannot load api scenario files from different repo?

Reply:
the 'fileRoot' here is passed with the autorest input-files root, which works as default fileRoot in this function.

If the fileRoot is configured at readme like https://github.com/Azure/azure-sdk-tools/pull/3613/files#diff-8eea1895b2ccb3c9bdca13e3bee2b50ec4a46e69999824ad2fe0665fa49b6889R84, the loader will use that configured value.
So the user can choose to use scenarios in another repo with this configuration.